### PR TITLE
Update to spyder3 so that plugin works again

### PR DESCRIPTION
**Pull request by jitseniesen**

### CODE REVIEW

File: spyder_unittesting/unittesting.py [hunk 1]
```diff
@@ -37,7 +38,7 @@ def get_plugin_title(self):
 
     def get_plugin_icon(self):
         """Return widget icon"""
-        return get_icon('profiler.png')
+        return ima.icon('profiler')
```
Comment (jitseniesen) [inline spyder_unittesting/unittesting.py]: It seems a bit strange to me to use the profiler icon here

**Comment by jitseniesen** [comment]
I should say that I changed the default back from nosetest to py.test so that I can check that everything works. I'm happy to remove this change from the Pull Request.

**Comment by jitseniesen** [comment]
I rebase and updated the pull request. @Nodd any comments or do you want me to merge it?

File: spyder_unittesting/unittesting.py (hunk 1 above)
Comment (Nodd) [inline spyder_unittesting/unittesting.py]: This is a leftover, this plugin has no icon yet. It's not really in a finished state...

File: spyder_unittesting/widgets/unittestinggui.py [hunk 2]
```diff
@@ -166,9 +150,7 @@ def __init__(self, parent):
                            self.start_button, self.stop_button, browse_button,
                            self.collapse_button, self.expand_button):
                 widget.setDisabled(True)
-            text = _(
-                '<b>Please install the <a href="%s">line_profiler module</a></b>'
-                ) % WEBSITE_URL
+            text = _('<b>Please install the unittesting module</b>')
```
Comment (Nodd) [inline spyder_unittesting/widgets/unittestinggui.py]: It should not be unittesting here but pytest I think. Again, this was not finished...

**Comment by Nodd** [comment]
Sorry, I wasn't watching the repo so I didn't get the notification. I initially planned to add an option to choose between pytest and nostests (or other when the are added).
Thanks for the PR.

**Pull request merged**